plugins/modules/dcnm_network.py

```diff
@@ -735,7 +741,7 @@ def get_want(self):
         if not self.config:
             return
 
-        for net in self.config:
+        for net in self.validated:
```

Comment:
What is the reason for this change?

Reply:
the earlier one is wrong, we verify/validate the self.config and store it in self.validated and  we should use this instead of self.config in the get_want. This is right approach has all the playbook config needs to be validated or filled in with default values in self.validated and that should be used in get_want. This is available in inventory/interface modules.